test/net/sourceforge/kolmafia/request/ScrapheapRequestTest.java

```diff
@@ -65,7 +66,10 @@ public void parseCPUUpgrades() throws IOException {
           "robot_potions",
           "robot_hp2"
         };
+    Arrays.sort(expected);
+
     var actual = Preferences.getString("youRobotCPUUpgrades").split(",");
+    Arrays.sort(actual);
```

Comment:
Instead of sorting the arrays, you could use hamcrest matchers:
```
import static org.hamcrest.MatcherAssert.assertThat;
import static org.hamcrest.Matchers.arrayContainingInAnyOrder;

assertThat(actual, arrayContainingInAnyOrder(expected));
```

Given that the number of expected matches is so large I'm not sure that reads any better than sorting them, but it could be nicer in other cases with fewer expected matches.

Reply:
OK.

Reply:
Done.